DQM/SiStripMonitorClient/interface/SiStripDcsInfo.h

```diff
@@ -86,6 +86,7 @@ class SiStripDcsInfo: public edm::EDAnalyzer {
     MonitorElement* DcsFractionME;
     int TotalDetectors;
     std::vector<uint32_t> FaultyDetectors;
+    std::map<uint32_t,uint32_t> NLumiDetectorIsFaulty;
```

Comment:
Please ignore my earlier comment here which is now deleted - misunderstanding. I switched to 16 bits for the second argument but cannot get an unordered_map declaration to compile in the header. Is 'map' sufficient, or should I pursue a larger code change to get 'unordered_map' to work?